Content.Server/Chemistry/ReagentEffects/Polymorph.cs

```diff
@@ -0,0 +1,37 @@
+using Content.Server.Mind.Components;
+using Content.Server.Polymorph.Components;
+using Content.Server.Polymorph.Systems;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Polymorph;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+public sealed partial class Polymorph : ReagentEffect
+{
+    /// <summary>
+    ///     What polymorph prototype is used on effect
+    /// </summary>
+    [DataField("prototype")] public string PolymorphPrototype { get; set; }
```

Comment:
Custom type serializer

Reply:
Sort of blindfired here with `customTypeSerializer:typeof(PrototypeIdSerializer<PolymorphPrototype>)`